src/extension.ts

```diff
@@ -49,7 +49,9 @@ export let ZOWE_TMP_FOLDER;
 export let USS_DIR;
 export let DS_DIR;
 export let ISTHEIA: boolean = false; // set during activate
-export const FAV_SUFFIX = "_fav";
+export const CONTEXT_PREFIX = "_";
```

Comment:
@Colin-Stone, question...
Should we document that `_` is not allowed when providing context information by extenders?
I'm curious what will happen with the `getBaseContext` function if extenders had a `_` as part of their context name. 
I don't think we would be able to properly update favorites.

Reply:
I will document this as an addendum to the contributors guide

Reply:
I think this is only an issue for uss_session. I can either globally rename it as ussSession or put in a specific test in the getBaseContext for it. The rename is the right thing to do (28 replaces) but the quick fix has less impact. Which would you prefer @zFernand0 ?

Reply:
I think I'd go for consistency here 😞 Sorry.
So, Change it to `ussSession` if it's not too disruptive.

Reply:
Thinking about this some more I will do option 2 for now and option1 should be a separate issue which will be implemented after Katelyn has merged her big PR

Reply:
Good catch! :) 
Thanks!